app/src/main/java/org/gdg/frisbee/android/activity/MainActivity.java

```diff
@@ -106,7 +103,6 @@ public void onCreate(Bundle savedInstanceState) {
         mSlidingTabLayout.setDistributeEvenly(true);
         mSlidingTabLayout.setOnPageChangeListener(this);
 
-        mViewPagerAdapter = new MyAdapter(this, getSupportFragmentManager());

```

Comment:
Initializing the adapter with nothing does nothing. It is initialized in only 1 place now which is `initChapters`